src/arrays.js

```diff
@@ -1,61 +1,126 @@
 const getNthElement = (index, array) => {
   // your code here
+  if (index > array.length - 1) {
+    index -= array.length;
+  }
+  return array[index];
```

Comment:
As before the `// your code here` comment should be removed

This could be simplified using the remainder operator:
```suggestion
  return array[index % array.length];
```